bashbrew/bashbrew.sh

```diff
@@ -413,6 +414,7 @@ while [ "$#" -gt 0 ]; do
 						"$docker" rmi -f "$namespace/$repoTag-$dateStamp"
```

Comment:
That removes `$repoTag`, what about `$repoTag-$dateStamp`? Or do we not create a datestamped version without the namespace?